src/main/java/com/celements/docform/DocFormCommand.java

```diff
@@ -196,7 +196,7 @@ XWikiDocument updateDocFromParam(XWikiDocument xdoc, XWikiDocument tdoc,
 
   private XWikiDocument setDocField(XWikiDocument tdoc, DocFormRequestParam param) {
     DocFormRequestKey key = param.getKey();
-    ClassField<String> field = xDocClassDef.getField(key.getFieldName(), String.class).orNull();
+    ClassField<String> field = xDocClassDef.getField(key.getFieldName(), String.class).orElse(null);
```

Comment:
why at all converting the optional to null? 'field' is only used twice in the following lines. This could be easily done with the optional instead of a null check.